src/Pages/Explanation.tsx

```diff
@@ -19,6 +19,23 @@ class Explanation extends React.Component {
                 <div className="content">
```

Comment:
Was findest du von der Idee, dass wir die Erklärung in ein modal packen?

So geht aktuell meine Eingabe verloren, wenn ich auf die Erklärung klicke

Reply:
und dann trotzdem auf der Erklärungsseite oder? finde die Modal Idee gut

Reply:
Ja gute Idee 👍